src/System.Linq.Expressions/tests/System.Linq.Expressions.Tests.csproj

```diff
@@ -8,22 +8,62 @@
+    -->
+    <IsInterpreting>false</IsInterpreting>
     <DefineConstants Condition=" '$(IsInterpreting)' != 'true' ">$(DefineConstants);FEATURE_COMPILE</DefineConstants>
     <DefineConstants Condition=" '$(FeatureInterpret)' == 'true' ">$(DefineConstants);FEATURE_INTERPRET</DefineConstants>
+    <IncludeDefaultReferences Condition="'$(TargetGroup)'=='uapaot'">false</IncludeDefaultReferences>
   </PropertyGroup>
   <!-- Default configurations to help VS understand the configurations -->
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uapaot-Windows_NT-Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uapaot-Windows_NT-Release|AnyCPU'" />
   <ItemGroup>
     <TargetingPackExclusions Include="System.Linq.Expressions" />
     <ReferenceFromRuntime Include="System.Linq.Expressions" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'uapaot'">
+    <ReferenceFromRuntime Include="System.Private.CoreLib" />
+    <ReferenceFromRuntime Include="xunit.core" />
+    <ReferenceFromRuntime Include="Xunit.NetCore.Extensions" />
+    <ReferenceFromRuntime Include="xunit.assert" />
+    <ReferenceFromRuntime Include="xunit.abstractions" />
+    <ReferenceFromRuntime Include="xunit.performance.core" />
+    <ReferenceFromRuntime Include="xunit.performance.api" />
+    <ReferenceFromRuntime Include="System.Runtime" />
+    <ReferenceFromRuntime Include="System.Collections" />
+    <ReferenceFromRuntime Include="System.Reflection.Primitives" />
+    <ReferenceFromRuntime Include="Microsoft.CSharp" />
+    <ReferenceFromRuntime Include="System.Runtime.Extensions" />
+    <ReferenceFromRuntime Include="System.Private.Uri" />
+    <ReferenceFromRuntime Include="System.Text.RegularExpressions" />
+    <ReferenceFromRuntime Include="System.Diagnostics.Debug" />
+    <ReferenceFromRuntime Include="System.ObjectModel" />
+    <ReferenceFromRuntime Include="System.Threading.Thread" />
+    <ReferenceFromRuntime Include="System.Threading.Tasks" />
+    <ReferenceFromRuntime Include="System.Linq" />
+    <ReferenceFromRuntime Include="System.Linq.Queryable" />
+    <ReferenceFromRuntime Include="System.Reflection" />
+    <ReferenceFromRuntime Include="System.IO.FileSystem" />
+    <ReferenceFromRuntime Include="System.Runtime.InteropServices.RuntimeInformation" />
+    <ReferenceFromRuntime Include="System.Runtime.InteropServices" />
+    <ReferenceFromRuntime Include="System.Private.Interop" />
+    <ReferenceFromRuntime Include="netstandard" />
+    <Reference Include="System.Reflection.Emit.ILGeneration" />
```

Comment:
Actually not yet but there is no easy way to remove that reference for now so it's fine to merge and we'll fix this later